### ISSUE TITLE
Finetune dataset

### ISSUE THREAD

**Issue by laura-wang** [imisra/shuffle-tuple#3]
Hello, Imisra.
In your paper,  you said that the pretrain dataset is the training split 1 of UCF101, and the fine-tune dataset is **test split 1 of UCF101**. 
Could you please tell me that do you use the **label** of   **test split 1 of UCF101** to fine-tune? If not, then **what is the ground-truth**? The tuple verification? 
Thanks a lot!
